apps/design-system-docs/package.json

```
@@ -35,9 +37,14 @@
 		"@docusaurus/tsconfig": "^3.7.0",
 		"@docusaurus/types": "^3.7.0",
 		"@types/react": "^18.3.21",
+		"@wordpress/base-styles": "^5.23.0",
 		"@wordpress/browserslist-config": "^6.23.0",
+		"sass": "^1.54.0",
 		"typescript": "^5.8.3"
 	},
+	"optionalDependencies": {
+		"sass-embedded": "^1.89.0"
+	},
```

I think I'd expect to have just `sass-embedded` as a required dependency and not at all depend on `sass`.

```suggestion
		"sass-embedded": "^1.89.0"
		"typescript": "^5.8.3"
	},
```

---

I tried that first, but then yarn will log warnings due to the dep being [declared](https://github.com/rlamana/docusaurus-plugin-sass/blob/8335cba73bd7375d2151d3982ce89f494b243b6d/package.json#L12) in `docusaurus-plugin-sass`.

Then I noticed that `sass-loader` itself [recommends](https://github.com/webpack-contrib/sass-loader#implementation) this "optionalDependencies" setup for compatibility reasons anyway.

---

TIL! That's interesting. I guess it makes sense, though unfortunate to take the burden of an extra dependency for an exceedingly unlikely edge case 😬